groovy/postInit/mod/GregTech.groovy

```diff
@@ -199,24 +199,63 @@ crafting.addShaped("pig_iron_tiny_pile_manual", metaitem('dustTinyPigIron'), [
         [null, metaitem('nuggetPigIron')]
 ]);
 
+//Steam Piston
+
 crafting.addShaped("gregtech:steam_piston", metaitem('steam.piston'), [
     [ore('plateBronze'), ore('plateBronze'), ore('plateBronze')],
     [ore('pipeTinyFluidBronze'), ore('stickBronze'), ore('stickBronze')],
     [ore('pipeTinyFluidBronze'), ore('craftingToolHardHammer'), ore('gearSmallBronze')]
 ])
 
+mods.gregtech.assembler.recipeBuilder()
+        .inputs(ore('plateBronze') * 3)
+        .inputs(ore('stickBronze') * 2)
+        .inputs(metaitem('gearSmallBronze') * 1)
+        .inputs(ore('pipeTinyFluidBronze') * 2)
+        .outputs(metaitem('steam.piston'))
+        .duration(100)
+        .EUt(30)
+        .buildAndRegister();
+
+//Steam Motor
+
 crafting.addShaped("gregtech:steam_motor", metaitem('steam.motor'), [
     [ore('plateBronze'), ore('stickBronze'), ore('plateBronze')],
     [metaitem('gearSmallBronze'), ore('stickBronze'), metaitem('gearSmallBronze')],
     [metaitem('steam.piston'), ore('stickBronze'), metaitem('steam.piston')]
 ]);
 
+mods.gregtech.assembler.recipeBuilder()
+        .inputs(ore('plateBronze') * 2)
+        .inputs(ore('stickBronze') * 3)
+        .inputs(metaitem('gearSmallBronze') * 2)
```

Comment:
gearSmallBronze also has a oredict entry, you should use that instead of the metaitem